dev/wlp-jakartaee-transform/rules/jakarta-messaging.properties

```diff
@@ -0,0 +1,3 @@
+javax.jms.ConnectionFactory=jakarta.jms.ConnectionFactory
+javax.jms.Destination=jakarta.jms.Destination
+javax.jms.Queue=jakarta.jms.Queue
```

Comment:
I note that elsewhere in this set of changes, a file has been changed solely to add a trailing newline. By that token, does this also need one?

Reply:
We can ignore the newline.  Other file(s) indicate a newline difference because I either added or removed a newline while resolving merge conflicts.